### PR TITLE
[CIT-473] fix the apt-plugin install with version

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
## Proposed changes

Code changes fix the issue that was returning the version of the apt-plugin when the apt-plugin is used to install a package with version number. For example "sudo /etc/tedge/sm-plugins/apt install Module  --version xyz"

Also updated the test to use version option to install a module.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactorings that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: sm/plugins/tedge_apt_plugin/src/main.rs [hunk 1]
```diff
@@ -15,6 +15,7 @@ pub enum PluginOp {
     /// Install a module
     Install {
         module: String,
+        #[structopt(short = "v", long = "--module-version")]
```
Comment (didier-wenzek) [inline sm/plugins/tedge_apt_plugin/src/main.rs:18]: The exact same thing must be done for the `remove` sub-command.

Comment (PradeepKiruvale) [reply sm/plugins/tedge_apt_plugin/src/main.rs:18]: Added

**Review by didier-wenzek — COMMENTED** [review]
Is there no test impacted by the change on the remove command?

File: tests/PySys/apt_plugin/environment_apt_plugin.py [hunk 2]
```diff
@@ -31,15 +31,13 @@ def plugin_cmd(
             args.append(argument)
 
         if version:
-            # TODO The version is specified but there is a name clashs
```
Comment (abelikt) [inline tests/PySys/apt_plugin/environment_apt_plugin.py:34]: Yes, away with that line, there is a typo :)

File: tests/PySys/apt_plugin/environment_apt_plugin.py [hunk 3]
```diff
@@ -31,15 +31,13 @@ def plugin_cmd(
...
-            # TODO Clarify CIT-473
-            # args.append("--version")
+            args.append("--module-version")
```
Comment (abelikt) [inline tests/PySys/apt_plugin/environment_apt_plugin.py:34]: I believe it should be still "--version" here and not "--module-version"

File: sm/plugins/tedge_apt_plugin/src/main.rs (hunk 1 above)
Comment (abelikt) [reply sm/plugins/tedge_apt_plugin/src/main.rs:18]: For me it looks good. I hope the UX is correct now (-v and --module-version).

File: tests/PySys/apt_plugin/environment_apt_plugin.py (hunk 3 above)
Comment (abelikt) [reply tests/PySys/apt_plugin/environment_apt_plugin.py:34]: Ok, I misunderstood this one. It's OK then

**Review by abelikt — APPROVED** [review]
Looks good for me

**Pull request merged**